sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/batch/TransactionalBatchResponse.java

```diff
@@ -0,0 +1,514 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.batch;
```

Comment:
should probably also be in an implementation package - see comment above the structure (public surface area vs. implementation package) needs to be looked at all-up.

Reply:
This is returned in case user uses TransactionalBatch. Can move few heavy functions to internal utilities/classes?

Reply:
Have segregated the public APIs in Data classes rather than a mix of data and function.